whois-api/src/test/java/net/ripe/db/whois/api/nrtmv4/NrtmClientServiceTestIntegration.java

```diff
@@ -349,12 +349,56 @@ public void should_throw_exception_invalid_source_notification_file()  {
     @Test
     public void should_throw_exception_delta_file_not_found()  {
         final Response response = createResource("TEST-NONAUTH/nrtm-delta.1.TEST-NONAUTH.60b9e8c4e4891411be.json")
-                .request(MediaType.APPLICATION_JSON)
+                .request(MediaType.APPLICATION_JSON).header("X-Forwarded-Proto", "HTTPS")
                 .get(Response.class);
```

Comment:
Can you create a abstract method that adds the header instead. You don't have to add in every call

Reply:
done